dotcom-rendering/cdk/lib/renderingStack.ts

```diff
@@ -118,6 +122,94 @@ export class RenderingCDKStack extends CDKStack {
 			ttl: Duration.hours(1),
 		});
 
+		const latencyMetric = new Metric({
+			dimensionsMap: {
+				LoadBalancer: ec2app.loadBalancer.loadBalancerFullName,
+				TargetGroup: ec2app.targetGroup.targetGroupFullName,
+			},
+			metricName: 'TargetResponseTime',
+			namespace: 'AWS/ApplicationELB',
+			period: Duration.seconds(30),
```

Comment:
How did we pick 30s?

Reply:
Yes, as @cemms1 says we used this to get faster feedback while testing. We could change it to 60 seconds to match the current prod latency alarm config if we don't want to make too many changes at once. We'll be monitoring closely anyway and make any necessary adjustments after this gets merged!

Reply:
I was thinking quicker! Breaking news may happen super quickly 😜 

Keep it at 30s for now.